packages/page-accounts/src/modals/Create.tsx

```diff
@@ -316,9 +320,9 @@ function Create ({ className = '', onClose, onStatusChange, seed: propsSeed, typ
                   onChange={_selectSeedType}
                   options={seedOpt}
                 />
-                <CopyButton
+                < CopyButton
                   className='copyMoved'
-                  isMnemonic
+                  type={seedType === 'bip' ? 'mnemonic' : seedType === 'raw' ? isEthereum ? 'private key' : 'seed' : 'development seed'}
```

Comment:
```suggestion
                  type={seedType === 'bip' ? t<string>('mnemonic') : seedType === 'raw' ? isEthereum ? t<string>('private key') : 'seed' : t<string>('raw seed')}
```

Just made it translatable. (and s/development/raw/)